example_deploy.sh

```diff
@@ -11,68 +10,71 @@ kubectl apply -f - <<EOF
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: ${APP}
-  namespace: ${NAMESPACE}
+  name: petclinic-without-attach
   labels:
-    app: ${APP}
+    app: petclinic-without-attach
+    service: petclinic-without-attach
   annotations:
     deployment.kubernetes.io/revision: "$revision"
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: ${APP}
+      app: petclinic-without-attach
```

Comment:
I think it'd be better to use ${APP} for templating. The APP variable value could be `petclinic` and the templating can be done in this line:

```suggestion
      app: ${APP}-without-attach
```